defaults/azure_config.yaml

```diff
@@ -14,9 +14,9 @@ azure_image_loader: 'OpenLogic:CentOS:7_9:latest'
 azure_image_monitor: 'OpenLogic:CentOS:7_9:latest'
 
 availability_zone: 'a'
-#root_disk_size_monitor: 50  # GB, remove this field if default disk size should be used
-#root_disk_size_db: 30  # GB, increase root disk for larger swap (maximum: 16G)
-#root_disk_size_loader: 30  # GB, Increase loader disk in order to have extra space for a larger swap
+azure_root_disk_size_monitor: 50  # GB, remove this field if default disk size should be used
+azure_root_disk_size_db: 30  # GB, increase root disk for larger swap (maximum: 16G)
+azure_root_disk_size_loader: 30  # GB, Increase loader disk in order to have extra space for a larger swap
```

Comment:
we have such settings for aws and gce - and we use different values there too (for db).

there's also some issue, that there's no such param like `gce_root_disk_size_loader` while it is used.

I created task for unification (https://trello.com/c/KIM3UpMF) - but it must be discussed if we want this and what values to store.

Reply:
well, each node type (monitor, DB and loader) shall have their own values for it, and AFAIU it will be an integer of GB...
and much better to have an unified parameter for it..